go/libraries/doltcore/sqle/dtables/unscoped_diff_table.go

```diff
@@ -0,0 +1,227 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dtables
+
+import (
+	"io"
+	"sort"
+
+	"github.com/dolthub/go-mysql-server/sql"
+
+	"github.com/dolthub/dolt/go/libraries/doltcore/diff"
+	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
+	"github.com/dolthub/dolt/go/libraries/doltcore/env/actions"
+	"github.com/dolthub/dolt/go/libraries/doltcore/sqle/index"
+)
+
+// UnscopedDiffTable is a sql.Table implementation of a system table that shows which tables have
+// changed in each commit, across all branches.
+type UnscopedDiffTable struct {
+	ddb  *doltdb.DoltDB
+	head *doltdb.Commit
+}
+
+// NewUnscopedDiffTable creates an UnscopedDiffTable
+func NewUnscopedDiffTable(_ *sql.Context, ddb *doltdb.DoltDB, head *doltdb.Commit) sql.Table {
+	return &UnscopedDiffTable{ddb: ddb, head: head}
+}
+
+// Name is a sql.Table interface function which returns the name of the table which is defined by the constant
+// LogTableName
+func (dt *UnscopedDiffTable) Name() string {
+	return doltdb.DiffTableName
+}
+
+// String is a sql.Table interface function which returns the name of the table which is defined by the constant
+// DiffTableName
+func (dt *UnscopedDiffTable) String() string {
+	return doltdb.DiffTableName
+}
+
+// Schema is a sql.Table interface function that returns the sql.Schema for this system table.
+func (dt *UnscopedDiffTable) Schema() sql.Schema {
+	return []*sql.Column{
+		{Name: "commit_hash", Type: sql.Text, Source: doltdb.LogTableName, PrimaryKey: true},
```

Comment:
It doesn't matter because this is only a result set, but for doc purposes, you need another key col than just commit_hash. I think it's actually (commit_hash, table_name), right?

I would also put table name as the second column for this reason

Reply:
Ah! Good catch! Fixed now. ✅